modules/apps/29-fee/keeper/migrations.go

```diff
@@ -23,6 +23,7 @@ func NewMigrator(keeper Keeper) Migrator {
 
 // Migrate1to2 migrates ibc-fee module from ConsensusVersion 1 to 2
 // by refunding leftover fees to the refund address.
+// TODO: should this be removed?
```

Comment:
Let's remove this todo and evaluate all migrations in https://github.com/cosmos/ibc-go/issues/7234

```suggestion
```